doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -3,29 +3,7 @@ HOST: http://orion.lab.fiware.org/
 
 # FIWARE-NGSI-Simple
 
-<!--
- Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
-
- This file is part of Orion Context Broker.
-
- Orion Context Broker is free software: you can redistribute it and/or
- modify it under the terms of the GNU Affero General Public License as
- published by the Free Software Foundation, either version 3 of the
- License, or (at your option) any later version.
-
- Orion Context Broker is distributed in the hope that it will be useful,
- but WITHOUT ANY WARRANTY; without even the implied warranty of
- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
- General Public License for more details.
-
- You should have received a copy of the GNU Affero General Public License
- along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
-
- For those usages not covered by this license please contact with
- iot_support at tid dot es
--->
-
-FIWARE-NGSI Simple API.
```

Comment:
To be coherent, we should apply the same change in the cookbook .apib, that also includes the FIWARE Open Specification statement.